airflow/dag_processing/processor.py

```diff
@@ -530,7 +530,13 @@ def manage_slas(cls, dag_folder, dag_id: str, session: Session = NEW_SESSION) ->
                 for callback in callbacks:
                     cls.logger().info("Calling SLA miss callback %s", callback)
                     try:
-                        callback(dag, task_list, blocking_task_list, slas, blocking_tis)
+                        callback(
+                            dag=dag,
+                            task_list=task_list,
+                            blocking_task_list=blocking_task_list,
+                            slas=slas,
+                            blocking_tis=blocking_tis,
+                        )
```

Comment:
Can we also confirm that callbacks work, not just notifiers, because this looks like a breaking change?
You can use this callback for sla:
```python
def my_sla_callback(dag, task_list, blocking_task_list, slas, blocking_tis):
    print(blocking_task_list)
```

Reply:
The logs are at $AIRFLOW_HOME/logs/scheduler/latest/PROJECT/DAG_FILE.py.log, see http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/administration-and-deployment/logging-monitoring/callbacks.html.

Reply:
@ephraimbuddy Varified it working with `my_sla_callback`.

Reply:
@ephraimbuddy Also, I think this will be a breaking change only if someone is using `args` in a callback to get the parameters passed. But if we want to be on the safe side we should consider it as a breaking change. 
![Screenshot 2023-06-15 at 7 17 52 AM](https://github.com/apache/airflow/assets/13021213/ad0452c5-22ab-4b32-878a-e1e81386392a)

Reply:
We should work it in a way that it's not a breaking change